src/wiringX.c

```diff
@@ -425,6 +428,118 @@ int wiringXSPISetup(int channel, int speed) {
 	return -1;
 }
 
+int wiringXSerialOpen(char *device, int baud) {
+	struct termios options;
```

Comment:
Shouldn't we make the `data bit`, `stop bit`, `parity` and `flow control` configurable. Maybe passing a struct as the second arguments insteaed of only the baud:
```
typedef struct wiringXSerial_t {
   unsigned int baud;
   unsigned int databits;
   unsigned int flowcontrol;
   unsigned int stopbit;
   unsigned int parity;
} wiringXSerial_t;
```